src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter((x) => x.sex === 'm');
+
+  if (century) {
+    men = men.filter(onePeople =>
+      century === Math.ceil(onePeople.died / 100)
+    );
+  }
```

Comment:
you can use ternary operatop and combine teo filers into one, in case, when you have century